package.json

```diff
@@ -25,5 +27,26 @@
   "bugs": {
```

Comment:
I can't comment on the exact line since it wasn't changed, but you should add a `contributors` section and put your name in it. :)